src/mumble/LookConfig.cpp

```diff
@@ -105,6 +107,20 @@ LookConfig::LookConfig(Settings &st) : ConfigWidget(st) {
 	
 }
 
+void LookConfig::updateFontInfo(QLabel *label, const QFont &font) {
+	label->setText(QString::fromLatin1("%1, %2pt").arg(font.family()).arg(font.pointSize()));
+}
+
+void LookConfig::updateFontInfo() {
+	updateFontInfo(s);
+}
+
+void LookConfig::updateFontInfo(const Settings &r) {
```

Comment:
I think `r` is a bad choice for a variable representing a setting. I'd also prefer a more verbose name. Something like `settings` I think would be ideal.